Makefile

```diff
@@ -1,6 +1,6 @@
 ## DO NOT EDIT!
 # This file was provisioned by OpenTofu
-# File origin: https://github.com/aetheric-oss/tofu-github/tree/main/src/modules/vars/templates/nuxt/all/Makefile.tftpl
+# File origin: https://github.com/aetheric-oss/tofu-github/tree/main/src/modules/varstemplates/nuxt/all/Makefile.tftpl
```

Comment:
vars/templates

Reply:
Hm, interesting, should be added automatically, wonder why only this one is missing the '/'.